packages/vue-client/src/assets/base.css

```diff
@@ -93,7 +93,6 @@
   }
 }
 
-/* TODO: Check why this is here, can we remove this? */
```

Comment:
Does this mean you figured it out?

Reply:
I've tested around a bit, and realised 2 things:
1.) We can view these as changes to the "default styles" of css, and I think these are good.
2.) Components can still overwrite these styles if need be.

I added this TODO when I added some detailed rules, and the markdown is displayed differently than I expected. For the rules it might still be worth to overwrite this, but it's fine for now imo.

Reply:
Makes sense, thanks for the explanation!